docs/advanced-usage/compute-backends/kubernetes/index.md

```diff
@@ -108,3 +100,87 @@ You can set `kubernetes_memory_limit` in every `rule` under `resources`.
         "docker://python:2.7-slim"
     ...
 ```
+
+## Job timeout
+
+When the job exceeds the specified time limit, it will be terminated by Kubernetes and marked as failed.
+
+To set the job timeout, you can specify `kubernetes_job_timeout` in the specification of **each workflow step**.
+Time is measured in **seconds**.
+
+Read more about the expected values and timeout logic on [Kubernetes official documentation](https://kubernetes.io/docs/concepts/workloads/controllers/job/#job-termination-and-cleanup)
+
+You can configure the `steps` in the respective specifications for Serial, Yadage, CWL and Snakemake workflow engines.
+
+For **Serial**, you can set `kubernetes_job_timeout` in every `step` of workflow specification.
```

Comment:
I would say let's end the paragraph with colon (`:`) which will better indicate that the code block below belongs to this phrase.  You can also change the `kubernetes_memory_limit` parts above to be the same...